applications/h264/h264_endoscopy_tool_tracking/python/h264_endoscopy_tool_tracking.py

```diff
@@ -0,0 +1,362 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from argparse import ArgumentParser
+
+try:
+    from holoscan.conditions import AsynchronousCondition, CountCondition, PeriodicCondition
+except ImportError as e:
+    raise ImportError(
+        "This example requires Holoscan SDK >= 2.1.0 so AsynchronousCondition is available."
+    ) from e
+from holoscan.core import Application
+from holoscan.gxf import load_extensions
+from holoscan.operators import FormatConverterOp, GXFCodeletOp, HolovizOp
+from holoscan.resources import (
+    BlockMemoryPool,
+    CudaStreamPool,
+    GXFComponentResource,
+    MemoryStorageType,
+    UnboundedAllocator,
+)
+
+from holohub.lstm_tensor_rt_inference import LSTMTensorRTInferenceOp
+from holohub.tensor_to_video_buffer import TensorToVideoBufferOp
+
+# Enable this line for Yuam capture card
+# from holohub.qcap_source import QCAPSourceOp
+from holohub.tool_tracking_postprocessor import ToolTrackingPostprocessorOp
+
+
+class VideoDecoderResponseOp(GXFCodeletOp):
+    def __init__(self, fragment, *args, **kwargs):
+        super().__init__(fragment, "nvidia::gxf::VideoDecoderResponse", *args, **kwargs)
+
+
+class VideoDecoderRequestOp(GXFCodeletOp):
+    def __init__(self, fragment, *args, **kwargs):
+        super().__init__(fragment, "nvidia::gxf::VideoDecoderRequest", *args, **kwargs)
+
+
+class VideoDecoderContext(GXFComponentResource):
+    def __init__(self, fragment, *args, **kwargs):
+        super().__init__(fragment, "nvidia::gxf::VideoDecoderContext", *args, **kwargs)
+
+
+class VideoReadBitstreamOp(GXFCodeletOp):
+    def __init__(self, fragment, *args, **kwargs):
+        super().__init__(fragment, "nvidia::gxf::VideoReadBitStream", *args, **kwargs)
+
+
+class VideoWriteBitstreamOp(GXFCodeletOp):
+    def __init__(self, fragment, *args, **kwargs):
+        super().__init__(fragment, "nvidia::gxf::VideoWriteBitstream", *args, **kwargs)
+
+
+class VideoEncoderResponseOp(GXFCodeletOp):
+    def __init__(self, fragment, *args, **kwargs):
+        super().__init__(fragment, "nvidia::gxf::VideoEncoderResponse", *args, **kwargs)
+
+
+class VideoEncoderContext(GXFComponentResource):
+    def __init__(self, fragment, *args, **kwargs):
+        super().__init__(fragment, "nvidia::gxf::VideoEncoderContext", *args, **kwargs)
+
+
+class VideoEncoderRequestOp(GXFCodeletOp):
+    def __init__(self, fragment, *args, **kwargs):
+        super().__init__(fragment, "nvidia::gxf::VideoEncoderRequest", *args, **kwargs)
+
+
+class EndoscopyApp(Application):
+    def __init__(self, data):
+        """Initialize the endoscopy tool tracking application"""
+        super().__init__()
+
+        # set name
+        self.name = "Endoscopy App"
+
+        if data == "none":
```

Comment:
I guess you are using "none" as a string from the argument parsing below, but seems more Pythonic to also support `None`
```suggestion
        if (data is None) or (data == "none"):
```